### PR TITLE
Change base url for API '/api/v1/'

### PR DESCRIPTION
**Pull request by michael-lark**
Any future major versions that break backward compatibility will get new names.

### CODE REVIEW

File: sql-server.py [hunk 1]
```diff
@@ -767,8 +767,8 @@ def safe_arg(request_arg):
 
 
 # Add all REST definitions here
-api.add_resource(Case, '/cases/<case_id>')
-api.add_resource(CaseList, '/cases')
+api.add_resource(Case, '/api/v1/cases/<case_id>')
+api.add_resource(CaseList, '/api/v1/cases')
```
Comment (caesar2164) [inline sql-server.py:771]: let's change these urls to `/api/cases/...`

**Review by caesar2164 — CHANGES_REQUESTED** [review]

**Comment by caesar2164** [comment]
@michael-lark you'll also have to change/squash your commits and make the message a bit nicer

File: sql-server.py (hunk 1 above)
Comment (michael-lark) [reply sql-server.py:771]: Well, from what I've read on APIs it is good form to have the version of your API included either in the url or in the header which is why I added the version. Then if at some point we publish a new version of the spec that breaks backward compatibility we can call it v2 and leave v1 also running.

Comment (caesar2164) [reply sql-server.py:771]: ok, agreed!

**Review by caesar2164 — APPROVED** [review]

**Pull request merged**